The build system includes its one check for available CMake build types.
This check is incorrect as it doesn't include the None build type.

diff -upr emscripten-fastcomp-1.39.0.orig/CMakeLists.txt emscripten-fastcomp-1.39.0/CMakeLists.txt
--- emscripten-fastcomp-1.39.0.orig/CMakeLists.txt	2020-02-22 00:06:01.828125245 +0100
+++ emscripten-fastcomp-1.39.0/CMakeLists.txt	2020-02-22 00:06:29.265827242 +0100
@@ -267,7 +267,7 @@ endif()
 string(TOUPPER "${CMAKE_BUILD_TYPE}" uppercase_CMAKE_BUILD_TYPE)
 
 if (CMAKE_BUILD_TYPE AND
-    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL)$")
+    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(NONE|DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL)$")
   message(FATAL_ERROR "Invalid value for CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
 endif()
 
